deploy_pulp3/pulp3_systemd/tasks/main.yml

```diff
@@ -0,0 +1,29 @@
+- name: Install Systemd unit files
+  # The template allows us to specify the platform python environment.
+  template:
+      src: "templates/{{ item }}.j2"
+      dest: "/etc/systemd/system/{{ item }}.service"
+  with_items:
+    - pulp_celerybeat
+    - pulp_resource_manager
+    - pulp_worker@
+- name: Create /var/cache/pulp directory
+  file:
+    path: /var/cache/pulp
+    state: directory
+    owner: "{{ pulp_user }}"
+    group: "{{ pulp_user }}"
+
+- name: reload systemd units
+  command: systemctl daemon-reload
```

Comment:
This can be dropped in favor of the `systemd` module's `daemon_reload` argument. This would also require using the `systemd` module instead of the `service` module, below.